build-customization.rst

```diff
@@ -0,0 +1,72 @@
+.. post:: Nov 10, 2022
+   :tags: announcement, feature, builds
+   :author: Manuel
+   :location: BCN
+
+.. meta::
+   :description lang=en:
+      We released a new config key (``build.commands``) to specify user-defined commands
+      which allows users to overwrite the build process completely.
+
+
+Overwrite the build process completely
```

Comment:
@ericholscher in a draft for a tweet in Front you mentioned that you would like to use something different than "Overwrite" for this. So, I'm pinging you here in case you have a better suggestion.

Reply:
```suggestion
Override the build process completely with build.commands
```

We call it override'ing in the docs, which is more correct: https://docs.readthedocs.io/en/stable/build-customization.html#override-the-build-process